README.md

```diff
@@ -42,13 +43,7 @@ Build deployable web page. This will generate the production web site.
 awestruct -P production
 ```
 
-Upload a web page to github.io. If everything went ok, web page will be updated automatically.
-```
-rsync -Pr --protocol=28 --delete-after _site/* jbosstm@filemgmt.jboss.org:www_htdocs/jbosstm/
-```
-
-
-If you did any changes on the development branch. Go back and push them to GitHub.
+To upload the (new) production web site, run the following command. Please, do not forget to set up the variable `RSYNC_HOST` before running the command. If everything went OK, the new web site is updated automatically.
 ```
-git push origin develop
+rsync -Pr --protocol=28 --delete-after _site/* jbosstm@${RSYNC_HOST}:www_htdocs/jbosstm/
```

Comment:
The bash syntax `${RSYNC_HOST}` seems out of place in a README. Other options could inlcude `jbosstm@<rysnc hostname>` ...

Reply:
It makes sense. I will replace the bash syntax with `<...>`